lib/ansible/modules/system/firewalld.py

```diff
@@ -656,6 +802,11 @@ def main():
     timeout = module.params['timeout']
     interface = module.params['interface']
     masquerade = module.params['masquerade']
+    chain = None if not module.params['chain'] else module.params['chain']
```

Comment:
It reads better the other way around:
```suggestion
    chain = module.params['chain'] if module.params['chain'] else None
```
But what is the purpose here ? To catch when the user provides an empty string ?